modyn/supervisor/internal/triggers/trigger_datasets/dataloader_info.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+
+
+# TODO(366): Unify with similar classes in trainer_server and evaluator
```

Comment:
I think that might be the wrong issue number. Might this be 275 instead?

Reply:
275 is about a common Dataset class, not DataLoaderInfo. I can create an issue about a common DataLoaderInfo.